libraries/pbrlib/genglsl/lib/mx_environment_fis.glsl

```diff
@@ -67,3 +67,5 @@ vec3 mx_environment_irradiance(vec3 N)
     vec3 Li = mx_latlong_map_lookup(N, $envMatrix, 0.0, $envIrradiance);
     return Li * $envLightIntensity;
 }
+
+#define MX39_USING_ENVIRONMENT_FIS
```

Comment:
Those defines are the only modifications on 1.38 library code. Allows updated versions to be enabled as required since the FresnelData struct is different for `mx39_` code.